### ISSUE TITLE
All mod not working

### ISSUE THREAD

**Issue by Svpenthopia** [olli-e/ISRT#10]
After NWI new 1.11 Hotifx #5.  All mod not working.  Can you please help check whose issue is this?  NWI or ISRT rcon tool.  I don't think its ISRT issue.

**Comment by olli-e** [comment]
Do you mean all mods are not working, or is there a mod called "All"? Can you give me a hint/link to which mod you are referring? Thanks in advance!

**Comment by Svpenthopia** [comment]
> Do you mean all mods are not working, or is there a mod called "All"? Can you give me a hint/link to which mod you are referring? Thanks in advance!

All my mod.  ISMCarmory_legacy,Hardcore,WelcomeMessage,JoinLeaveMessage,MoreAmmo.  And all my mod map.  Thanks

**Comment by olli-e** [comment]
Is it possible for you to temporary change your rcon pw and send me your server data, IP, Query and RCOn port incl. PW, so I can check on your server and what I see? Unfortunately I don't have a test server anymore. Please do not post your data here, but send it to my e-mailaddress madman@isrt.info Thx

**Comment by Svpenthopia** [comment]
> Is it possible for you to temporary change your rcon pw and send me your server data, IP, Query and RCOn port incl. PW, so I can check on your server and what I see? Unfortunately I don't have a test server anymore. Please do not post your data here, but send it to my e-mailaddress [madman@isrt.info](mailto:madman@isrt.info) Thx

I had changed only the Rcon Port to 2017 on one batch file.  The other batch file no changes also the same issue.  It was working all along for 1 week till today after the nwi update then I have this problem.

**Comment by olli-e** [comment]
Is it possible for you to temporary change your rcon pw and send me your server data, IP, Query and RCOn port incl. PW, so I can check on your server and what I see? Unfortunately I don't have a test server anymore. Please do not post your data here, but send it to my e-mailaddress [madman@isrt.info](mailto:madman@isrt.info) Thx

**Comment by Svpenthopia** [comment]
On Friday, 11 February 2022, 21:17:21 GMT+8, Olli E ***@***.***> wrote:  
 
 


Is it possible for you to temporary change your rcon pw and send me your server data, IP, Query and RCOn port incl. PW, so I can check on your server and what I see? Unfortunately I don't have a test server anymore. Please do not post your data here, but send it to my e-mailaddress ***@***.*** Thx

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Svpenthopia** [comment]
> On Friday, 11 February 2022, 21:17:21 GMT+8, Olli E ***@***.***> wrote: Is it possible for you to temporary change your rcon pw and send me your server data, IP, Query and RCOn port incl. PW, so I can check on your server and what I see? Unfortunately I don't have a test server anymore. Please do not post your data here, but send it to my e-mailaddress ***@***.*** Thx — Reply to this email directly, view it on GitHub, or unsubscribe. Triage notifications on the go with GitHub Mobile for iOS or Android. You are receiving this because you authored the thread.Message ID: ***@***.***>

I had sent you the log file

**Comment by olli-e** [comment]
I don't need the logfile - don't you understand what I ask you for? I need access to the server, otherwise I cannot find the error, because nobody else has reported this, this is most probably an error on your side and I need to track it down on your end.

**Comment by Svpenthopia** [comment]
> 

Oh you mean I turn on my server and you can access in to find the error, am i correct?

**Comment by Svpenthopia** [comment]
> Yes, but I need the server data - ip-address, query and rcon port as well as password

ip - 192.168.10.115, Query Port - 27131, Rcon port - 27017  Pw - 8556.   Thanks

**Comment by olli-e** [comment]
You cannot send me an internal IP-Address - that doesn't work, because it is not routed in the internet....you need an official IP address.

**Comment by Svpenthopia** [comment]
> Yes, but I need the server data - ip-address, query and rcon port as well as rcon password (change it temporaril

> You cannot send me an internal IP-Address - that doesn't work, because it is not routed in the internet....you need an official IP address.
 you mean the ipv6 address?

**Comment by olli-e** [comment]
No, the official IPv4 address on your internet connection - the one you would need to connect from outside of your home

**Comment by Svpenthopia** [comment]
> No, the official IPv4 address on your internet connection - the one you would need to connect from outside of your home

Then this is the one.  192.168.10.115

**Comment by olli-e** [comment]
No, that's an internal ip-address which is not reachable over the internet. Will not work that way - I guess you don't know about port forwarding and NAT on your router, so this is most probably not a way to got. I this case, I cannot help you.

**Comment by Svpenthopia** [comment]
> No, that's an internal ip-address which is not reachable over the internet. Will not work that way - I guess you don't know about port forwarding and NAT on your router, so this is most probably not a way to got. I this case, I cannot help you.
you meant public ipv6?

**Comment by Svpenthopia** [comment]
> > No, that's an internal ip-address which is not reachable over the internet. Will not work that way - I guess you don't know about port forwarding and NAT on your router, so this is most probably not a way to got. I this case, I cannot help you.
> > you meant public ipv6?

116.88.88.143

**Comment by olli-e** [comment]
Query is working but RCON doesn't work via port 27017 - port forwarding and NAT for TCP/27017 established? Is it the right port for RCON?

**Comment by Svpenthopia** [comment]
> 



> Query is working but RCON doesn't work via port 27017 - port forwarding and NAT for TCP/27017 established? Is it the right port for RCON?

The actual recommended Rcon port is 27015.  This is the 2nd server using 27017.  Remember you told me to control individual server it must have a different Rcon Port.  So thats what i did for this 2nd server.  I use 27017.  

Nevermind I turn on the other 1st server with the actual rcon port 27015, Query Port -27231, Password -8556  now.    You can log in that one

**Comment by olli-e** [comment]
The same - rcon doesn't work, Query works.

**Comment by Svpenthopia** [comment]
> The same - rcon doesn't work, Query works.

Rcon - 27015 is correct

**Comment by Svpenthopia** [comment]
> Try it yourself with the official IP until it works with rcon - then I can continue to help

ok ...I update you soon

**Comment by olli-e** [comment]
It is possible, but very unlikely, otherwise I would have hundreds of people complaining about the tool not working. You have a networking problem here and you have shown a....let's say....improvable network skill by assigning two different daemons the same port using TCP - so before saying a tool isn't working, you ahve to exclude all other things that could be wrong on your side. I tried to find a solution, but you are not providing me with a working rcon port on your server. So there you have the problem - please don't blame the tool. Just try another tool and you'll find the same problems - that would prove that you have something not working properly on your side. I can't and won't tell you about how networks work, that's you job as a server admin. I just provide a tool that's working for more than 300 people, but why not for you....because the tool has an error? Very unlikely.

Well, I will leave you with that thought and say sorry, that I was not able to help you. Please use a different tool!
Thanks and I write this with all respect, but the facts are the facts and all you bring up are assumptions based on assumptions.
Madman

**Comment by Svpenthopia** [comment]
Hi,  Thanks for trying to help.  Actually from the very first msg I already said I don't think its ISRT Rcon issue.  I was hoping you could help confirm my thoughts.  Thanks again Ollie-e.  Appreciate it.

**Comment by olli-e** [comment]
No problem, you can always come back to me, when you have a working configuration for your rcon - as long as rcon doesn't work at all, my tool is no good. Once rcon works, I can see if the tool produces any error, before that, not. By working I mean you can at least show the map list via rcon using the official IP-address.
Sorry, but this is the minimum that has to work, otherwise, as said, my tool does not work at all, regardless of anything else.
Cheers,
Madman

**Comment by Svpenthopia** [comment]
Sure.  As usual.  It’s NWI issue.  They always screwed other things up after any huge update.  I suspected it.  Because after I updated four of my personal server.  All four server has this issue.  I am looking into it.  The link discussion.  Thank you so much for helping.  Even go beyond even its not the tool issue.   Really appreciate your kindness.  Thanks buddy. 👍

**Comment by Svpenthopia** [comment]
After I did the settings what Stuermer taught us to do.  It works now.  Again.  Thanks for you help Ollie-e.  Appreciate your kindness.  Have a good day  
    On Saturday, 12 February 2022, 09:18:07 GMT+8, Yusuf ***@***.***> wrote:  
 
 Sure.  As usual.  It’s NWI issue.  They always screwed other things up after any huge update.  I suspected it.  Because after I updated four of my personal server.  All four server has this issue.  I am looking into it.  The link discussion.  Thank you so much for helping.  Even go beyond even its not the tool issue.   Really appreciate your kindness.  Thanks buddy. 👍  

Sent from my iPhone

On 12 Feb 2022, at 2:32 AM, Olli E ***@***.***> wrote:



﻿


You may also want to have a look here, since the mods are broken, because they changed something in the config:

https://steamcommunity.com/app/581320/discussions/0/4510921146048002669/

But, again, it's a server issue, not a tool issue.

—
Reply to this email directly, view it on GitHub, or unsubscribe.
Triage notifications on the go with GitHub Mobile for iOS or Android.
